package.json

```diff
@@ -39,7 +39,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lerna run --concurrency 1 --stream precommit"
+      "pre-commit": "lerna run --concurrency 1 --stream precommit",
+      "pre-push": "lerna run --concurrency 1 --stream nightwatch:browserstack"
```

Comment:
This command runs nightwatch:browserstack agains production. Why would we do this pre-push?